src/layout/src/components/App/AppContentWrapper.test.js

```diff
@@ -99,7 +100,7 @@ describe('AppContentWrapper', () => {
 
         test('"max-width", "min-width", "width" defaults', () => {
           const wrapper = mount(<AppContentWrapper />);
-          expect(wrapper.find(AppContentInnerWrapper)).toHaveStyleRule('max-width', `${APP_CONTAINER_MAX_WIDTH}px`);
+          expect(wrapper.find(AppContentInnerWrapper)).toHaveStyleRule('max-width', undefined);
```

Comment:
This doesn't seem necessary.

Reply:
True, but it'll be nice if this fails in the case that someone changes it in the future because we specifically don't want `max-width` to be set.